warehouse/accounts/forms.py

```diff
@@ -121,11 +121,12 @@ def validate_password(self, field):
                     raise wtforms.validators.ValidationError(
                         _("The password is invalid. Try again.")
                     )
-            except TooManyFailedLogins:
+            except TooManyFailedLogins as err:
                 raise wtforms.validators.ValidationError(
                     _(
                         "There have been too many unsuccessful login attempts. "
-                        "Try again later."
+                        f"You have been locked out for {err.resets_in.total_seconds()} "
```

Comment:
So here the units are in seconds, rather than minutes -- but I think there was a suggestion to use the [humanize](https://pypi.org/project/humanize/) module made as well here, given that e.g. if the timeout period is 600 seconds, seeing "10 minutes" is a bit nicer than seeing "600 seconds".

Perhaps after the test suite passes it's worth giving a shot at adding that module as a dependency and seeing whether you can wire it up here.